evap/evaluation/management/commands/send_reminders.py

```diff
@@ -11,16 +11,17 @@
 
 
 class Command(BaseCommand):
-    args = '<kind of jobs>'
-    help = 'Runs updates/tasks based on time events'
+    help = 'Runs updates based on time events'
+    args = 'daily'
+
+    def handle(self, *args, **options):
+        assert (len(args) > 0 and args[0] == 'daily')
```

Comment:
daily can be removed now, right? it was used to differentiate between the two tasks which isn't necessary now.